yaml/screenconnect.yaml

```diff
@@ -56,7 +56,17 @@ Artifacts:
   - File: C:\ProgramData\ScreenConnect Client*\user.config
     Description: ScreenConnect client user configuration
     OS: Windows
-  EventLog: []
+  EventLog:
+  - EventID: 7045
+    ProviderName: ["ScreenConnect", "ScreenConnect Client (<hex string>)"]
```

Comment:
ProviderName should be "Service Control Manager" according to the main repository's samples

Reply:
Removed this since it is a standard Windows event log, not specific to ScreenConnect.